js_modules/dagster-ui/packages/ui-core/src/assets/AssetsCatalogTable.tsx

```diff
@@ -62,7 +62,13 @@ export function useCachedAssets({
 
   return {cacheManager};
 }
-const requery = () => [{query: ASSET_CATALOG_TABLE_QUERY, fetchPolicy: 'no-cache' as const}];
+const requery = () => [
+  {
+    query: ASSET_CATALOG_TABLE_QUERY,
+    variables: {limit: DEFAULT_BATCH_LIMIT},
+    fetchPolicy: 'no-cache' as const,
+  },
+];
```

Comment:
I actually think this was a mistake because the assets come from the query in `useAllAssets` which doesn't see the result of this requery so I think it just gets dropped into the void

Reply:
Hmm interesting... so we need to use a different mechanism to refresh and this `requery` thing needs to go away? Will take a look

Reply:
Confirmed that this was not doing what it should anymore -- Instead of triggering a requery this way, I'm updating this to call `refreshState.refetch()` and confirmed that behaves as expected.